tools/parse_baselines/providers/types.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The commits says there is an implementation for vsock baselines as well, but it misses from this commit.

Reply:
it's not actually missing, it's just that the `NetworkTcpDataParser` is used for both vsock and tcp tests.
I'll rename it to IperfDataParser

Reply:
done, renamed to `Iperf3DataParser`